setup.py

```diff
@@ -13,6 +13,7 @@
 
 python_requirements = '>=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*'
 requirements = ['configparser',
+                'packaging',
```

Comment:
You should use the same version as specified in constraints:

```suggestion
                'packaging~=18.0',
```